lib/configuration/configuration.ts

```diff
@@ -16,6 +16,16 @@ export interface ActionOnFile {
   watchAssetsMode: boolean;
 }
 
+export interface SwcCliOptions {
```

Comment:
```suggestion
interface SwcCliOptions {
```

Reply:
Export of `SwcCliOptions` has been removed